### PR TITLE
Set pointer from the repo/README to polymer site

### PR DESCRIPTION
**Pull request by SerkanSipahi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# layout
+layout
+=========
+See the [page](https://www.polymer-project.org/docs/polymer/layout-attrs.html) for more information
```
Comment (ssorallen) [inline README.md]: Can you make the link text more informational?

```
See the [layout attributes documentation](...) for more information.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# layout
+layout
+=========
```
Comment (ssorallen) [inline README.md:2]: Add a blank line after the `======` as done in other element repos like `core-icon`: https://raw.githubusercontent.com/Polymer/core-icon/master/README.md

Comment (SerkanSipahi) [inline README.md:2]: @ssorallen done :)

**Comment by ssorallen** [comment]
Great thanks a lot.

**Pull request merged**